CONTRIBUTING.md

```diff
@@ -72,25 +72,21 @@ will be easily spotted in pull request. It will also ensure that OpenTelemetry
 doesn't expose APIs outside of the library primary concerns like a generic
 helper methods.
 
-#### How to enable and configure
+#### How to configure
 
 * Create a folder in your project called `.publicApi` with the frameworks that
   as folders you target.
 * Create two files called `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt`
    in each framework that you target.
-* Add the following lines to your csproj:
-
-```xml
-<ItemGroup>
-  <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Shipped.txt" />
-  <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Unshipped.txt" />
-</ItemGroup>
-```
-
 * Use
    [IntelliSense](https://docs.microsoft.com/visualstudio/ide/using-intellisense)
    to update the publicApi files.
 
+It is enabled by default. Add `<RunApiCompat>false</RunApiCompat>` to your
+`.csproj` file to temporarily disable it. This should only be done when
+bypassing api check while developing locally or until a stable version of the
```

Comment:
all components in this repo should have api check irrespective of if they are stable.. I'd say - document this as a quick-hack-for-local-testing, and initial PR about the API shape itself.